frontend/appflowy_tauri/src/appflowy_app/components/document/_shared/TextInput.hooks.ts

```diff
@@ -161,13 +161,13 @@ function setSelection(editor: ReactEditor, currentSelection: TextSelection) {
   const { path, offset } = currentSelection.focus;
   // It is possible that the current selection is out of range
   const children = getDeltaFromSlateNodes(editor.children);
-  if (children[path[1]].insert.length < offset) {
+  const index = path[1];
```

Comment:
What happen if the length of the path is zero

Reply:
The path is not allowed to be zero or null. Because the slate node is a `Two-dimensional array`, its path must be [number, number]

Reply:
Got it. it would be better if you add comments there.